bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescriptionRegistry.java

```diff
@@ -165,4 +182,52 @@ public ConfigDescription getConfigDescription(URI uri) {
         return getConfigDescription(uri, null);
     }
 
+    /**
+     * Updates the config parameter options for a given URI and parameter
+     * <p>
+     * If more than one {@link ConfigOptionProvider} is registered for the requested URI, then the returned
+     * {@link ConfigDescriptionParameter} will contain the data from all providers.
+     * <p>
+     * No checking is performed to ensure that multiple providers don't provide the same options. It is up to
+     * the binding to ensure that multiple sources (eg static XML and dynamic binding data) do not contain overlapping
+     * information.
+     *
+     * @param uri
+     *            the URI to which the options to be returned (must not be null)
+     * @param locale
+     *            locale
+     * @return config description
```

Comment:
This line is about the return value, not a parameter. But you are right that a line for `@param parameter` is missing.